src/cases/components/allocate-role/allocate-role.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import { RoleAllocationCaptionText, RoleAllocationRadioText, RoleAllocationTitleText, RoleAllocationType } from '../../../cases/enums';
+
+@Component({
+  selector: 'exui-allocate-role',
+  templateUrl: './allocate-role.component.html'
+})
+
+export class AllocateRoleComponent implements OnInit {
+
+  @Input() public roleAllocation: RoleAllocationType;
+  @Input() public includeOther: boolean;
```

Comment:
would this best to move to src\work-allocation-2\components, if this is only used in work allocation 2.